File: 30 Days of Code/Day 10/sol.java
import java.io.*;
import java.util.*;

public class Solution {

    public static void main(String[] args) {
        /* Enter your code here. Read input from STDIN. Print output to STDOUT. Your class should be named Solution. */
        Scanner in = new Scanner(System.in);
        int dec = in.nextInt();
        Stack<Integer> st = new Stack<Integer>();
        int rm = 0;
        while(dec>0){
            rm = dec%2;
            dec = dec/2;
            st.push(rm);
        }
        
        int size = st.size();
        int[] array = new int[size];
        for(int i = 0 ; i < size; i++){
            array[i] = st.pop();
        }
     
        int countOne = 0;
        int maxOne =0;
        for(int j = 0 ; j < size ; j++){
            if(array[j] == 0){
                countOne = 0;  
            } 
            else{
                countOne++;
                maxOne = Math.max(maxOne,countOne);
            }
        }
        System.out.println(maxOne);
    }
}

File: 30 Days of Code/Day 9/sol.c
#include <stdio.h>
#include <string.h>
#include <math.h>
#include <stdlib.h>
int factorial(int n );
int main() {
    
    /* Enter your code here. Read input from STDIN. Print output to STDOUT */ 
    int inp;
    scanf("%d",&inp);
    
    printf("%d",factorial(inp));
    return 0;
}
int factorial(int n ){
   // int result =1;
    if(n <= 1) return 1;
    return n*factorial(n-1);
}
